src/js/game_logic.js

```diff
@@ -1,8 +1,28 @@
+const shuffle = (arr1) => {
+    let ctr = arr1.length
+    let temp
+    let index
+
+    while(ctr > 0) {
+        index = Math.floor(Math.random() * ctr)
+        ctr --
+        temp = arr1[ctr]
+        arr1[ctr] = arr1[index]
+        arr1[index] = temp
```

Comment:
So you can use deconstruction to avoid using temporary holders when switching variables.
```
aVar = 1
bVar = 2
[aVar, bVar] = [bVar, aVar]
```

Reply:
I will need to review this.